apps/web/src/components/TrendingSuggestionList/TrendingSuggestionList.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { useQuery } from 'react-query';
+import TrendingItemSuggestion from '../TrendingItemSuggestion/TrendingItemSuggestion';
+import './TrendingSuggestionList.scss';
+
+interface Item {
+  /** How we identify each item */
+  id: number;
+  /** How can we view the suggested post? */
+  URL: string;
+  /** Suggestion Title */
+  trendingSuggestionHeading: string;
+  /** Suggestion Summary */
+  trendingSuggestionSummary: string;
+  /** Specific forum the suggestion lives in */
+  targetSubreddit: {
+    /** Forum's Icon */
+    subRedditIcon: React.ReactElement;
+    /** Forum's title */
+    subRedditName: string;
+  };
+}
+
+interface Props {
+  customTestId?: string;
+}
+
+const TrendingSuggestionList = ({
+  customTestId = 'suggestion-list',
+}: Props): React.ReactElement => {
+  const { isLoading, data, isFetching, isFetched } = useQuery(
+    'suggested-items',
+    async () => {
+      const response = await fetch('../../Data/SuggestedItems.json');
```

Comment:
Additionally, instead of native `fetch` API I think we should switch to something more robust like Axios. That should help with the typing issue that you mentioned.

Reply:
Finally, the `async` function that actually fetches the data can be moved to a special API file, which will exist at the top-level, so in the same place as the component that will call `useQuery`, in this case a top-level `<TrendingSuggestions />` component. The Axios functions that fetch or post data would be written in a `TrendingSuggestions.api.ts` file, that way we can easily write tests against those functions.